src/staticanalysis/bot/static_analysis_bot/cli.py

```diff
@@ -100,6 +100,20 @@ def main(source,
         taskcluster_access_token,
     )
 
+    # Load queue service
+    queue_service = get_service(
+        'queue',
+        taskcluster_client_id,
+        taskcluster_access_token,
+    )
+
+    # TRASHME
+    from static_analysis_bot.revisions import ImprovementPatch
```

Comment:
Using this to test from Taskcluster tasks, [as we cannot run locally this code](https://docs.taskcluster.net/docs/reference/platform/taskcluster-queue/references/api#create-artifact)